include/use-case.h

```diff
@@ -342,6 +342,16 @@ int snd_use_case_get_list(snd_use_case_mgr_t *uc_mgr,
  *   - CaptureMasterType
  *      - type of the master volume control
  *      - Valid values: "soft" (software attenuation)
+ *   - CaptureChannelMap
+ *      - Remap channels using ALSA PCM channel mapping API notation
+ *        E.g. "2 3 0 1 -1 -1 -1 -1 -1 -1 -1" means, FL takes channel 2,
+ *        FR takes channel 3, RL takes channel 0, RL takes channel 1.
+ *   - CoupledMixers
+ *      - Mixer controls that should be changed together.
+ *        E.g. "Left Master,Right Master".
+ *   - EchoReferencePCM
```

Comment:
There is modifier for this https://github.com/alsa-project/alsa-lib/blob/master/include/use-case.h#L166 . The application may obtainm the PCM device name from this modifier.

Reply:
Got it, I will add this to my list of things to remove from CRAS.